decidim-assemblies/db/migrate/20241016101151_migrate_assembly_members_to_private_users.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+class MigrateAssemblyMembersToPrivateUsers < ActiveRecord::Migration[7.0]
```

Comment:
I have doubts about not making here an script, but since the PR is deleting the models it'll be more complicated to manage, so I think this is a good solution 👍🏾